.github/workflows/email.yml

```diff
@@ -29,6 +29,7 @@ jobs:
               echo "COMPARE_URL= $( echo '${{github.event.repository.html_url}}/compare/${{github.event.pull_request.base.sha}}...${{ github.event.pull_request.merge_commit_sha}}' )" >> $GITHUB_ENV      
               echo "LOG=$(echo '[Merge pull request] # ${{github.event.number}} from ${{github.event.pull_request.head.label}}' )" >> $GITHUB_ENV
               echo "FILES_CHANGED=$(echo '${{github.event.pull_request._links.html.href}}/files' )" >> $GITHUB_ENV
+              echo "SUBJECT= $(echo '[Chapel Merge]  ${{github.event.number}}  ${{github.event.pull_request.title}}' )" >> $GITHUB_ENV
```

Comment:
There are two spaces between the parts here, can it be just one?

Reply:
@mppf Yes, will remove a space in the next PR that I am working on to add some more formatting. Did not see the comment before merging.